src/config.ts

```diff
@@ -112,7 +112,7 @@ const DEFAULTS = o({
         "mdn":"https://developer.mozilla.org/en-US/search?q=",
         "gentoo_wiki":"https://wiki.gentoo.org/index.php?title=Special%3ASearch&profile=default&fulltext=Search&search=",
         "qwant":"https://www.qwant.com/?q=",
-
+        "reddit":"https://reddit.com/r/%s",
```

Comment:
I don't think most users typing `open reddit blah` expect to go to the blah subreddit. We could have a `subreddit` search provider, perhaps.

And, of course, you can add your own providers locally with `setsearchkeyword reddit whatever`